src/main/java/com/quiz/service/impl/UserQuizServiceImpl.java

```diff
@@ -93,4 +104,63 @@ public UserQuiz getUserQuizById(long userQuizId) {
     public List<UserQuiz> getAllUserQuiz() {
         return userQuizRepository.getAllUserQuizs();
     }
+
+    @Override
+    public String takeQuiz(long userId, int quizId) {
```

Comment:
This branch and my local computer are not the same, I will solve the problem as soon as possible.